wagtail/admin/templates/wagtailadmin/pages/search_results.html

```diff
@@ -11,14 +11,35 @@ <h2>
 
         {% search_other %}
 
-        {% include "wagtailadmin/pages/listing/_list_explore.html" with show_parent=1 %}
+        {% if pages.object_list.supports_facet %}
+            <nav class="listing-filter">
+                <h3 class="filter-title">{% trans "Content types" %}</h3>
```

Comment:
Unless anyone objects, I'll change this to "Page types" as I think that's a more meaningful term for editors.